spec/airport.spec.js

```diff
@@ -0,0 +1,174 @@
+import Airport from "../src/airport.js";
+import Plane from "../src/Plane.js";
+import Weather from "../src/Weather.js";
+
+describe('Airport Tests', () => {
```

Comment:
Consider incorporating mocking to isolate the class under test and test its behaviour independently from its dependencies (Plane & Weather).